### ISSUE TITLE
Status logged out

### ISSUE THREAD

**Issue by Bozo29** [nlrb/com.soundunited.heos#9]
Dear Ramon,
I can't connect to Heos, try everyting.
The Heos app (v 2.47.470) works perfect on my Marantz SR6013 (Homey Early 2016 v 8.0.3)
Install the D&M Heos again with the right login en password, everytime "Status logged out".
What do I wrong?

**Comment by nlrb** [comment]
Are you using this app or the official app? For me the login works. Note that logging in is only needed to get your stations/favorites.

**Comment by Bozo29** [comment]
Thanks for the quick response.

I use the D&M Heos app, this app.
I have read somewhere that you can stream a music URL (radio-URL) to the reciever with this app.
That's exectly what I want.
I have the IP protocol commands of the reciever, like MVUP (master volume up), SIAUX2 (select input source aux 2) etc.
Now I want this for Heos, to send a RAW command to the reciever with a Radio-URL string with the "Marantz" app from Marco van den Hout, if it's possible. But i can't find the commands for Heos.
TuneIn doesn't have all the radio channels that i want to have and it's not all in HD.

**Comment by nlrb** [comment]
There is a flow card to play a URL. You don't need to be logged in to an account for that.

**Comment by Bozo29** [comment]
Oke, but the problem is that when I go to Homey/apparaten/+/D&M Heos to search a reciever in the network, he finds nothing.
Is that because the app D&M Heos has the status logged out?

**Comment by nlrb** [comment]
No, that is unrelated. It just searches for devices on the network.

**Comment by Bozo29** [comment]
I apreciate your (quick) answers.
Then the question now is:
The official android Marantz app and the Heos app on my phone reconized the reciever in my network, why doesn't the D&M Heos app on Homey do it? Is it a port question in the router or something else?

**Comment by nlrb** [comment]
No, idea. Impossible to debug like this. But if both finding devices and login does not work, I’m wondering if Homey is on the same network.

**Comment by Bozo29** [comment]
Dear Ramon,
Yes, the Marantz and Homey are in the same network. The app from Marco van de Hout works well. I have contact Marantz about the IP protocol commands for Heos and I have try some things. But no succes.
I give up.
Thanx Ramon for all your help.
